src/Juvix/Compiler/Core/Transformation/LambdaLifting.hs

```diff
@@ -4,13 +4,11 @@ module Juvix.Compiler.Core.Transformation.LambdaLifting
   )
 where
 
-import Juvix.Compiler.Core.Data.InfoTableBuilder
+import Juvix.Compiler.Core.Pretty
```

Comment:
Minor: we don't need pretty printing here, beyond debugging.

Reply:
I wonder why ghc didn't complain about that... Anyway, it is fixed now